utils/main_test.go

```diff
@@ -15,11 +15,30 @@
 package utils
 
 import (
+	"crypto/rand"
+	"crypto/rsa"
+	"encoding/base64"
 	"os"
 	"path/filepath"
+	"strings"
 	"testing"
+
+	"golang.org/x/crypto/ssh"
 )
 
+func mkRandRSAPubKey(t *testing.T) string {
```

Comment:
Can you please move this function to a non OS specific go file and avoid the duplication?

Reply:
Created utils/test.go file to host this test specific util function. Please let me know if you'd prefer another place for this function